static/app/views/issueList/overview.tsx

```diff
@@ -296,13 +296,6 @@ class IssueListOverview extends React.Component<Props, State> {
       return query as string;
     }
 
-    if (
-      organization.features.includes('inbox') &&
-      organization.features.includes('inbox-tab-default')
-    ) {
-      return Query.FOR_REVIEW;
```

Comment:
Should it default to these now?

Reply:
nope, we were using this internally but are not going to change the default tab